aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -486,6 +486,13 @@ module aptos_framework::account {
         (signer, signer_cap)
     }
 
+    /// create the account for system reserved addresses
+    public(friend) fun create_framework_reserved_account(addr: address): (signer, SignerCapability) {
+        let signer = create_account_unchecked(addr);
+        let signer_cap = SignerCapability { account: addr };
+        (signer, signer_cap)
+    }
```

Comment:
this should make a check that it is between 0x1 and 0x10 or something...

Reply:
updated